scripts/Muon/GUI/ElementalAnalysis2/grouping_widget/ea_grouping_table_widget_presenter.py

```diff
@@ -9,10 +9,9 @@
 from Muon.GUI.ElementalAnalysis2.ea_group import EAGroup
 from mantidqt.utils.observer_pattern import GenericObservable
 from Muon.GUI.ElementalAnalysis2.grouping_widget.ea_grouping_table_widget_view import INVERSE_GROUP_TABLE_COLUMNS
-from Muon.GUI.Common import message_box
-
 
 MAXIMUM_NUMBER_OF_GROUPS = 20
+REBIN_NONE_OPTION = "0"
```

Comment:
Not for this PR, but next time you touch this code look at shifting this into a proper Python Enum class